store/build.xml

```diff
@@ -35,15 +35,16 @@
     <path refid="class.path"/>
     <pathelement location="${build.classes.dir}"/>
   </path>
-  <target name="imapd-control">
-    <if>
+
+  <target name="test.if.imapd.available">
+    <condition property="imapd.available">
       <available file="${zimbra.home.dir}/bin/zmimapdctl" type="file"/>
-      <then>
-        <exec executable="zmimapdctl">
-          <arg value="${action}"/>
-        </exec>
-      </then>
-    </if>
+    </condition>
+  </target>
+  <target name="imapd-control" unless="imapd.installed" depends="test.if.imapd.available">
```

Comment:
I'm guessing should be run `if="imapd.available"` instead of `unless="imapd.installed"`?